src/krisi/report/type.py

```diff
@@ -5,9 +5,10 @@
 import pandas as pd
 import plotly.graph_objects as go
 from dash import dcc
+from krisi.evaluate.type import MetricResult
 
 PlotFunction = Callable[
-    [pd.Series, pd.DataFrame, List[Union[pd.Series, pd.DataFrame, int, float]]],
+    [List[MetricResult], str, Optional[float]],
```

Comment:
I copied this from the function signature, the previous was incorrect, and resulted in a type error